### ISSUE TITLE
aria-role issues with global nav header menu list items

### ISSUE THREAD

**Issue by cosbyr** [Esri/global-nav#298]
Parent <ul> elements should have a role of "list" rather than "navigation" or keep role as "navigation" and add a unique aria-label attribute to each one
![80022462-f2176d00-8490-11ea-9e5f-5bb536bb6ab9](https://user-images.githubusercontent.com/11748268/83304664-195e0a00-a1b4-11ea-8534-4d96f7e4f172.png)
![80022672-428eca80-8491-11ea-80ae-d603ec6108c3](https://user-images.githubusercontent.com/11748268/83304668-1a8f3700-a1b4-11ea-8ec1-a8eb26891af8.png)
![80022873-897cc000-8491-11ea-8710-882fe53ffa7e](https://user-images.githubusercontent.com/11748268/83304673-1c58fa80-a1b4-11ea-91bb-ff4b82540d33.png)
